server/graphql/loaders/index.js

```diff
@@ -710,6 +710,15 @@ export const loaders = req => {
     }).then(results => sortResults(CollectiveIds, results, 'CollectiveId', [])),
   );
 
+  context.loaders.PaymentMethod.byIdIncludeDeleted = new DataLoader(keys => {
+    return models.PaymentMethod.findAll({
+      where: {
+        id: keys,
+      },
+      paranoid: false,
+    }).then(results => sortResultsSimple(keys, results, pm => pm.id, []));
+  });
```

Comment:
Quick reminder: this is only safe because we use the primary key ID in this query.
Hypothetically, we should not implement stuff like `byCollectiveIdIncludedDeleted` because our custom indexes exclude deleted records.